### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by NicoTuxx**
# What has changed ?

- [ ] Does this PR affect the User ?

### CODE REVIEW

File: oxidation/client/src/views/HomePage.vue [hunk 1]
```diff
@@ -113,19 +166,27 @@ async function canDismissModal(): Promise<boolean> {
...
-  position: absolute;
-  left: 0;
-  right: 0;
-  top: 50%;
-  transform: translateY(-50%);
+  height: 100%;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  // position: absolute;
```
Comment (TimeEngineer) [inline oxidation/client/src/views/HomePage.vue:173]: Remove comment

**Comment by FirelightFlagboy** [comment]
Hello, could you provide a better name for the title of the PR ?

For the icons/images, shouldn't we use `git-lfs` (that would reduce the size of the git folder) ? @touilleMan

**Comment by touilleMan** [comment]
@NicoTuxx what is this `oxidation/client/src/assets/images/Logo/__MACOSX` folder ?
the `__` as prefix plus the fact it only contains hidden (i.e. starting with a dot) files make it looks like macOS internals that should not be added to the project (like the infamous `.ds_store`)

**Comment by touilleMan** [comment]
And any idea why github is complaining about the files in the `__MACOSX ` folder ? are they valid file that can be read on you computer ?
image

**Pull request closed without merging**